### DISCUSSION TITLE
Plasma table

### DISCUSSION THREAD

**Discussion by mikf65** [phil-barrett/PicoCNC#14] [Q&A]
Hi I plan to use the picoCNC board for a plasma table which will need a torch height controller (probably DIY) system added to it.
I'm a bit new to this and would appreciate some input on possible issues with the plasma cutter and adding the THC. Is it possible to connect the THC to the picocnc board?
Thanks.

**Comment by JelmerV** [comment]
I am working on the same thing so I am joining this discussion.

For the plasma voltage, I soldered wires directly from the rectifier circuit within the plasma cutter. This must be done before the high-voltage transformer. But still, the HF-start generates a lot of noise.  A voltage divider drops the voltage from 100V to 3.3V, and then I added some inductors and capacitors for low-pass filtering. Not sure how this will work as will also influence the responsiveness of the THC.

Then for the THC I plan to use the [plasma plugin](https://github.com/grblHAL/Plugin_plasma). I am not sure how the connections are supposed to work for that but IO pins 26 or 27 should still be available and have an ADC. 27 would be AUX3 and is also what the plasma plugin seems to use.

I am interested in how this analog input is used on the PicoCNC board and what would be the best way to wire/configure the THC.

**Comment by mikf65** [comment]
Hi, good shout on the plasma plugin!! I missed that. I'm still in the planning stage and need to order the picoCNC board  ASAP as it will no doubt take a few weeks to reach NI.
